subprojects/docs/src/docs/userguide/img/taskInputsOutputs.graffle

```diff
@@ -0,0 +1,816 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
OK, let's keep it there for now. At the moment we only have a hand full of images anyway. In the long run I think I'd like to keep them separate.